presto-main/src/main/java/com/facebook/presto/operator/aggregation/GenericAccumulatorFactory.java

```diff
@@ -757,6 +819,10 @@ public void evaluateIntermediate(int groupId, BlockBuilder output)
             if (blockBuilders == null) {
                 checkState(rawInputs != null);
 
+                // Release memory held by markDistinctHash in delegate.
```

Comment:
let's avoid using the implementation detail comments like "markDistinctHash" in the interface invocation